File: 8 kyu/DNA to RNA Conversion/solution.js
function DNAtoRNA(dna){
  return dna.replace(/T/g, 'U');
}

_____________________________
const DNAtoRNA = dna => dna.replace(/T/g, 'U');

_____________________________
function DNAtoRNA(dna) {
  return dna.split("T").join("U");
}

_____________________________
function DNAtoRNA(dna) {
  var hold = ''
  for(var i =0;i<dna.length;i++) {
     if(dna[i]=="T") {
        hold+="U"
}
else{hold+=dna[i]}
}
return hold;
      
}

File: 8 kyu/DNA to RNA Conversion/solution.rb
def dna_to_rna(dna)
  dna.gsub('T','U')
end

_____________________________
def dna_to_rna(dna)
  dna.tr("T","U")
end

_____________________________
def dna_to_rna(dna)
 chars = dna.chars
 rna = ""
 chars.each do |c|
   c == "T" && c = "U"
   rna << c
 end
  rna
end

_____________________________
def dna_to_rna(dna)
  dna.gsub(/T/, 'U')
end

_____________________________
def dna_to_rna(dna)
  dna.chars.map { |char| char == "T" ?  char = "U"  : char}.join
end
